server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/measure/Measure.java

```diff
@@ -218,7 +245,9 @@ public UpdateMeasureBuilder setQualityGateStatus(QualityGateStatus qualityGateSt
      *
      * @throws NullPointerException if the specified {@link MeasureVariations} is {@code null}
      * @throws UnsupportedOperationException if the source measure already has a {@link MeasureVariations}
+     * @deprecated as only one period is now available. Use {@link #setVariation(double)} instead
      */
+    @Deprecated
     public UpdateMeasureBuilder setVariations(MeasureVariations variations) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)